critiquebrainz/frontend/forms/review.py

```diff
@@ -20,13 +21,32 @@ def __call__(self, form, field):
             raise ValidationError(self.message)
 
 
-# Loading supported languages
-languages = []
-for language_code in supported_languages:
+OTHER_LANGUAGES_CODES = supported_languages
+FREQUENTLY_USED_LANGUAGES_CODES = get_top_languages()
+for language in FREQUENTLY_USED_LANGUAGES_CODES:
     try:
-        languages.append((language_code, Locale(language_code).language_name))
+        OTHER_LANGUAGES_CODES.remove(language)
```

Comment:
Since it's required to put this code in a function, I'd suggest to name the variables in lowercase.